cms/templates/cms/home_page.html

```diff
@@ -34,6 +34,15 @@
   <meta property="og:image:height" content="630">
   <meta name="twitter:card" content="summary">
   <meta name="twitter:site" content="@MITxonedX">
+  <link href="https://plus.google.com/+MITX" rel="publisher">
```

Comment:
I've confirmed that MITx doesn't have a Google+ account. I hope we can just drop this line.